### PR TITLE
chore: Move saving of branding config to tenant config

### PR DESCRIPTION
**Pull request by ankitakinger**
## Description

Move saving of branding config to tenant config

#### PR fixes following issue(s)
Fixes # (issue number)

#### Type of change
- Chore (housekeeping or task changes that don't impact user perception)

#### How Has This Been Tested?
- [x] Manual
- [x] Jest
- [x] Cypress

#### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#speedbreakers-) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#areas-of-interest-)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [ ] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

File: app/client/src/ce/pages/AdminSettings/config/branding/index.tsx [hunk 1]
```diff
@@ -20,4 +20,5 @@ export const config: AdminConfigType = {
   icon: "pantone",
   component: BrandingPage,
   isFeatureEnabled: isBrandingEnabled(featureFlags),
+  needsUpgrade: true,
```
Comment (dipyamanbiswas07) [inline app/client/src/ce/pages/AdminSettings/config/branding/index.tsx:23]: This will add the locked icon to branding, do we want that?

**Review by dipyamanbiswas07 — APPROVED** [review]

File: app/client/src/ce/pages/AdminSettings/config/branding/index.tsx (hunk 1 above)
Comment (ankitakinger) [reply app/client/src/ce/pages/AdminSettings/config/branding/index.tsx:23]: yeah should be there, we need to show the same UI on the leftPane for all business features.

**Pull request merged**